guide/web-scene.md

```diff
@@ -0,0 +1,71 @@
+## X3D and Web Scene
+
+### Description
+
+Webots can export a world to an interactive 3D `HTML` page.
+This feature is especially useful for publishing Webots-created worlds on the Web.
+
+When doing so, an [X3D](http://www.web3d.org/x3d/what-x3d) file and a `HTML5` page are generated.
+
+The `X3D` file is containing the graphical information of the world.
+`X3D` is a modern `XML`-based file format for representing 3D computer graphics.
+It can be imported in many 3D softwares (for visualizing, modeling, etc.).
+
+The `HTML5` page is containing a Webots player which displays the `X3D` file
+as shown in [the figure bellow](#screenshot-of-a-web-scene-page-generated-by-webots).
+It can be open in any recent Web browser as documented in [the section bellow](#remarks-on-the-used-technologies-and-their-limitations).
+The navigation in the player is done using the mouse or the touch screen, similarly than the Webots navigation.
+
+%figure "Screenshot of a Web scene page generated by Webots"
+
+![screenshot-web-scene.png](images/screenshot-web-scene.png)
+
+%end
+
+
+### How to export a Web Scene
+
+Select the `File / Export HTML5 Model...` menu item and choose the target `HTML` file.
+Webots is asking to playback the resulting file in the default browser.
+
+**Note**:
+The `X3D` file and the required textures are exported in the same directory as the target `HTML` file.
+
+**Note**:
+The playback option may not work correctly depending on your default Web browser
+(cf. [section bellow](#remarks-on-the-used-technologies-and-their-limitations))
```

Comment:
Missing final '.'

same comment as before about brackets and cf.

typo: bellow -> below